Dockerfile

```diff
@@ -2,6 +2,11 @@
 FROM java:8u111-jdk-alpine
 MAINTAINER varkockova.a@gmail.com
 
+# to avoid libnative-platform.so missing bug in gradle
```

Comment:
It's not actually a bug - `libstdc++` is something that is present on almost all Linuxes but Alpine is very minimal...